### PR TITLE
change file param from hard code to config file

### PR DESCRIPTION
**Pull request by cw123**
### What problem does this PR solve?

Problem Summary: change file param from hard code to config file

### What is changed and how it works?

What's Changed: change file param from hard code to config file

How it Works: change file param from hard code to config file

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by cw123** [comment]
recheck

**Comment by cw123** [comment]
recheck

**Review by xu-chaojie — APPROVED** [review]

File: conf/mds.conf [hunk 1]
```diff
@@ -167,6 +167,12 @@ mds.copyset.scatterWidthFloatingPercentage=20
 #
 # curvefs的默认chunk size大小，16MB = 16*1024*1024 = 16777216
 mds.curvefs.defaultChunkSize=16777216
+# curvefs的默认segment size大小，1GB = 1*1024*1024*1024 = 1073741824
+mds.curvefs.defaultSegmentSize=1073741824
+# curvefs的默认最小文件大小，10GB = 10*1024*1024*1024 = 10737418240
+mds.curvefs.minFileLength=10737418240
+# curvefs的默认最大文件大小，20TB = 20*1024*1024*1024*1024 = 21990232555520
```
[the remark this answers is not in the document]
Comment (cw123) [reply conf/mds.conf:174]: The original configuration file comments are in Chinese

File: curve-ansible/roles/generate_config/templates/mds.conf.j2 [hunk 2]
```diff
@@ -175,6 +175,12 @@ mds.copyset.scatterWidthFloatingPercentage={{ mds_copyset_scatterwidth_floating_
 #
 # curvefs的默认chunk size大小，16MB = 16*1024*1024 = 16777216
 mds.curvefs.defaultChunkSize={{ chunk_size }}
+# curvefs的默认segment size大小，1GB = 1*1024*1024*1024 = 1073741824
+mds.curvefs.defaultSegmentSize={{ segment_size }}
+# curvefs的默认最小文件大小，10GB = 10*1024*1024*1024 = 10737418240
+mds.curvefs.minFileLength={{ min_file_length }}
+# curvefs的默认最大文件大小，20TB = 20*1024*1024*1024*1024 = 21990232555520
```
[the remark this answers is not in the document]
Comment (cw123) [reply curve-ansible/roles/generate_config/templates/mds.conf.j2:182]: The original configuration file comments are in Chinese

**Review by SeanHai — APPROVED** [review]

**Pull request merged**